pages/common/doctl-databases.md

```diff
@@ -0,0 +1,20 @@
+# doctl databases
+
+> Manage MySQL, Redis, PostgreSQL, and MongoDB database services.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/>.
+
+- List database clusters:
+
+`doctl databases list`
+
+- Display commands for managing database users:
+
+`doctl databases user`
+
+- Migrate a database cluster to a new region interactively:
+
+`doctl databases migrate`
+
+- Get details for a database cluster:
```

Comment:
```suggestion
- Get details for the current database cluster:
```

Optional change: `the current` may sound better.
Hint: as no argument provided for `get` subcommand `the current` fallback was suggested instead of `a specific`.